model/src/main/kotlin/config/PostgresStorageConfiguration.kt

```diff
@@ -44,5 +44,40 @@ data class PostgresStorageConfiguration(
      * The password to use for authentication.
      */
     @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
-    val password: String = ""
+    val password: String = "",
+
+    /**
+     * Connect using SSL.
+     * See: https://jdbc.postgresql.org/documentation/head/ssl-client.html
+     */
+    val ssl: Boolean = false,
+
+    /**
+     * The SSL mode to use, one of disable, allow, prefer, require, verify-ca or verify-full.
+     * See: https://jdbc.postgresql.org/documentation/head/ssl-client.html
+     */
+    val sslmode: String? = null,
```

Comment:
What's the difference between `ssl == false` and `sslmode == "disable"`? Can we drop the former explicit flag?

Reply:
I don't know why they offer the `disable` mode or if there is a difference not enabling ssl at all, but I would expect it to be the same. One the one hand I think it makes sense to copy the parameters of the JDBC driver here, on the other hand e.g. libpq does not have a `ssl` setting and you have to set the mode to `disable` there:
https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS

Also, the JDBC driver is a bit confusing, currently putting `ssl == false` would still enable SSL, because it is only checking for the existence of the parameter, they only encourage to set it to `true` because this might change in future.

If we want to mimic libpq we could always set the `ssl` parameter and require the user to explicitly disable SSL. And if the user does not set a value for `sslmode` we just use the default which currently is `verify-full`.

Reply:
I have updated the PR accordingly.

Reply:
And I have updated the `README` for the new options.